apps/docs/pages/guides/realtime/rate-limits.mdx

```diff
@@ -11,23 +11,48 @@ Supabase Realtime is a global cluster. We've implemented some rate limits to hel
 
 Rate limits are configurable per project and our cluster supports millions of concurrent connections. [Contact support](https://app.supabase.com/support/new) if these limits are causing issues.
 
-## Limits by Plan
+## Rate Limits by Plan
 
-Limits are set on the Free and Pro plans accordingly. See our [pricing page](https://supabase.com/pricing) for more information.
+Rate limits are different than usage limits. The usage limits for Free and Pro projects can be found on the See our [pricing page](https://supabase.com/pricing).
 
-Enterprise plans are billed on a usage basis. We still employ limiting on Enterprise plans. If you're on an Enterprise plan simply contact support and we'll increase your limits as needed.
+Rate limits protect our infrasructure. You can increase your rate limits by upgrading your plan and removing your spend cap.
```

Comment:
@chasers we should position this as rate limits exist to ensure you're not unknowingly overspending and to make sure that Realtime stays performant under expected usage.